release_template.md

```diff
@@ -0,0 +1,73 @@
+---
+name: Tracking issue
+about: Use this template for upcoming releases.
+title: "[Release Number] Release Checklist"
+---
+
+Target date:
+
+Release Candidate by DD/MM/YYYY
+
+Release by DD/MM/YYYY
+
+Release candidate
+-------------------
+- [ ] Verify that all relevant PRs have been merged to master.
+- [ ] Create a PR against master to bump version number, merge that PR
+- [ ] From the commit just before bumping the version, create a new branch `maint/<release version number>`
+- [ ] Make sure `maint/<release version number>` is included (somehow) in the following block in appveyor.yml 
+```
+branches:
+  only:
+    - ...
+```
```

Comment:
Ah just envisage does not exclude builds in the Appveyor setup, so the PR against the maintenance branches are probably always run.  We can remove this for now and see if this is needed later (I suspect not).

Reply:
Somehow this comment went to the file before the move to `.github/ISSUE_TEMPLATE`... I probably had the old (and too many) tabs opened when I commented, sorry about that :(